packages/sdk/package.json

```diff
@@ -44,6 +44,7 @@
     "@solarwinds-apm/histogram": "workspace:^",
     "@solarwinds-apm/lazy": "workspace:^",
     "@solarwinds-apm/module": "workspace:^",
+    "json-stringify-safe": "^5.0.1",
```

Comment:
`JSON.stringify` throws when it encounters a cyclical reference, this doesn't. Very simple package, hasn't changed in over 5 years, and maintained by the creator of NPM, so I don't have an issue depending on it.